frontend/src/console/components/CoursesManage/styles.scss

```
@@ -0,0 +1,80 @@
+@import "~styles/theme";
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@600&display=swap');
+
+.courses-page {
+    // Only put this because the page didn't pick up the right font family.
+    font-family: 'Montserrat';
+
+    h2 {
+        font-family: 'Montserrat' !important;
+        font-size: 30px;
+        font-weight: 600;
+        font-stretch: normal;
+        font-style: normal;
+        line-height: normal;
+        letter-spacing: normal;
+        color: #000000;
+    }
+
+    p {
+        margin-top: 20px;
+    }
+
+    .row {
+        margin-bottom: 5px;
+    }
+
+    .numberCircle {
+        border-radius: 50%;
+        width: 1.6em;
+        height: 1.6em;
+        padding: 0.20em;
+        background: #1e4c59;
+        color: #666;
+        text-align: center;
+
+        p {
+            margin-top: 0;
+            font-size: 0.8em;
+            color: white;
+        }
+    }
+
+    .list-items {
+        margin-top: 1.4em;
+    }
+
+    h6 {
+        display: inline;
+        color: $secondary-2;
+
+        .support-link {
+            margin-left: 5px;
+            text-decoration: none;
+            font-weight: bold;
+        }
+    }
+
+    .manageBtn {
+        display:flex;
+        width: 226px;
+        height: 40px;
+        margin-top: 50px;
+        padding: 7px 21px 10px 14px;
+        font-size: 19px;
+        font-weight: normal;
+        font-stretch: normal;
+        font-style: normal;
+        line-height: normal;
+        letter-spacing: normal;
+        text-align: justify;
+        color: #ffffff;
+        text-transform: none;
+        justify-content: space-evenly;
+
+        .fas {
+            padding-top: 2px;
+            padding-left: 8px;
+        }
+    }
```

Prefer the default styles here:
```suggestion
     button {
        i.fas {
            padding-left: 10px;
        }
    }
```

---

Not a fan of how this one looks but applied!